identifyyeastcells.py

```diff
@@ -0,0 +1,1146 @@
+            <li>Parameters are stored as a text in "Autoadapted parameters". If you found good parameters for your datasets
+            you can copy them to another pipeline.<\li>
+            </ul></dd>
+            </dl>
+            """%globals())
+
+        self.show_autoadapted_params = cps.Binary(
+            'Do you want to see autoadapted parameters?', False, doc="""
+            <i>(Used only when you want to specify advanced parameters)</i><br>
+            Use this option to display autoadapted parameters."""%globals())
+
+        self.autoadapted_params = cps.Text(text="Autoadapted parameters: ", value="[[0.0442, 304.45, 15.482, 189.40820000000002], [300, 10, 0, 18, 10]]", doc="""
+            <i>(Used only when you want to specify advanced and autoadapted parameters)</i><br>
+            Autoadapted parameters are pasted here from the "learning" preocedure. These parameters are used to characterize cell borders. 
+            Edit them only if you know what you are doing. If you found good parameters for your datasets
+            you can copy them to another pipeline.
+            """%globals())
+
+        self.should_save_outlines = cps.Binary(
+            'Retain outlines of the identified objects?', False, doc="Do you want to use objects outlines in modules downstream?")
+
+        self.save_outlines = cps.OutlineNameProvider(
+            'Name the outline image',"PrimaryOutlines", doc="""\
+            <i>(Used only if outlines are to be saved)</i><br>
+            You can use the outlines of the identified objects in modules downstream,
+            by selecting them from any drop-down image list."""%globals())
+
+    PRECISION_PARAMS_START = 20
+    PRECISION_PARAMS_END = 26
+
+    def settings(self):
+        return [self.input_image_name, 
+                self.object_name,
+                self.average_cell_diameter,
+                self.segmentation_precision,
+                self.maximal_cell_overlap,
+                self.background_image_name,
+                self.should_save_outlines,
+                self.save_outlines,
+                self.advanced_parameters,
+                self.background_brighter_then_cell_inside,
+                self.bright_field_image,
+                self.min_cell_area,
+                self.max_cell_area,
+                self.advanced_cell_filtering, 
+                self.background_elimination_strategy,
+                self.show_autoadapted_params,
+                self.autoadapted_params,
+                self.autoadaptation_steps,
+                self.ignore_mask_image_name,
+
+                self.specify_precision_details,
+                self.iterations,
+                self.seeds_border,
+                self.seeds_content,
+                self.seeds_centroid,
+                self.contour_points,
+                self.contour_precision,
+                self.weights_number,
+                ]
+
+    def visible_settings(self):
+        list = [self.input_image_name,
```

Comment:
`list` is a Python keyword. Best to use another variable name. `visible_settings` is what I usually use.